process/templates/PROJECT_NAME_graduation_stage.md

```diff
@@ -39,6 +39,10 @@ Projects should harden their build systems in accordance with the SLSA Framework
 When applicable, projects must have completed a security audit through a third party and addressed audit findings and recommendations.
   * "link to results of security audit"
 
+### Security Baseline
+
+The project must meet the "[Security Baseline - To Become Graduated](https://github.com/ossf/tac/blob/main/process/security_baseline.md#security-baseline---to-become-graduated)" requirements.
```

Comment:
We will also want to check for the "Once Incubating" requirements here: https://github.com/ossf/tac/blob/main/process/security_baseline.md#security-baseline---once-incubating

Reply:
Actually I realized that this ought to also list the requirements applying to the previous stage since the baseline doc doesn't make that clear. I listed them all.